lib/streamaggr/streamaggr.go

```diff
@@ -543,6 +563,21 @@ func (a *aggregator) push(tss []prompbmarshal.TimeSeries) {
 	promutils.PutLabels(labels)
 }
 
+func trackDroppedSample(ts *prompbmarshal.TimeSeries, actualTs, minTs int64, m *metrics.Counter) {
```

Comment:
I don't think that we should print log message for this case.

 If `keepInput`: true option is set, samples would be ingested any way, but not taken into account for aggregate calculation, so this message doesn't make much sense for user.

Reply:
Updated error message to make it more descriptive. I'd like to keep an error message as we have the same flow for discarding metrics with too old timestamps(outside of retention period) and it will make it easier to understand why metrics are discarded.